lib/cisco_node_utils/feature.rb

```diff
@@ -108,5 +108,23 @@ def self.vn_segment_vlan_based_enabled?
     end
 
     # ---------------------------
+    def self.fabric_frwd_anycast_mac
```

Comment:
Please keep these methods in alpha order (ie these methods should start at line 66)